### PR TITLE
Add Dockerfile and GitHub Actions Workflow

### PR DESCRIPTION
**Pull request by dellisd**

### CODE REVIEW

File: .github/workflows/build-image.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        env:
+          GHOST_CONFIG: ${{ secrets.GHOST_CONFIG }}
+        run: echo "$GHOST_CONFIG" > ghostConfig.json
+
+      - name: Build dist
+        run: docker build -t docker.pkg.github.com/cuhacking/main-website/main-website .
+
+      - name: Push image
+        if: success()
+        run: docker push docker.pkg.github.com/cuhacking/main-website/main-website:latest
```
Comment (FemmeQueenInSTEM) [inline .github/workflows/build-image.yml]: Are images in the registry displayed anywhere? As in, is there a way to verify that the image is in the registry on our own (before pulling from the openstack machine)?

Comment (dellisd) [reply .github/workflows/build-image.yml]: For the repo: https://github.com/cuhacking/main-website/packages/

For the whole org: https://github.com/orgs/cuhacking/packages

Comment (dellisd) [reply .github/workflows/build-image.yml]: It's a normal docker registry, so I believe it also supports all the same API endpoints as any other registry.

**Review by walsquared — APPROVED** [review]
So far so good 👍🏿

**Pull request merged**